.github/workflows/release-preview.yml

```diff
@@ -0,0 +1,152 @@
+# Matomo release action for automated PREVIEW releases
+#
+# Required GitHub secrets:
+#
+# GPG_CERTIFICATE  |  ASCII armored or Base64 encoded GPG certificate that is used to create the signatures for the archives
+# GPG_CERTIFICATE_PASS  |  Passphrase of the GPG key
+
+name: Build preview release
+
+permissions:
+  actions: read # required for the tests job
+  checks: none
+  contents: write  # required to create tag and release
+  deployments: none
+  issues: read # required for the tests job
+  packages: none
+  pull-requests: read # required for the tests jobs
+  repository-projects: none
+  security-events: none
+  statuses: none
+
+on:
+  # TODO: remove manual dispatch after testing and enable cron
+  workflow_dispatch:
+    branches:
+      - 5.x-dev
+  #schedule:
+  #  - cron: '0 1 * * *' # 1am daily
```

Comment:
Should we change that before merging or leave it in, so we can test that manually before having it run as cron?

Reply:
With the recent changes branching changes I would say we merge it as `workflow_dispatch` and run it once manually before updating it to a schedule. Just so we don't have to wait for the first scheduled run to see if any of the recent repository changes has an unforeseen impact :thinking: